### PR TITLE
DPC-1168: As an internal user I want a pop up when I'm trying to disable/delete org/user

### PR DESCRIPTION
**Pull request by Sun-Mountain**
**Why**

Add a pop up that asks the internal user to confirm that they are sure they want to delete the org from the api.

Add a pop up to confirm internal user wants to delete a user account.

This will inhibit user error by not allowing them to immediately delete an org or user account.

**What Changed**

Added confirm pop ups for deleting a user and an organization. Add confirm pop up for disabling an organization from the Sandbox API.

**Choices Made**

Screen Shot 2020-05-01 at 3 58 55 PM
Screen Shot 2020-05-01 at 3 59 49 PM
Screen Shot 2020-05-01 at 3 59 34 PM

**Tickets closed**:

[DPC-1168](https://jiraent.cms.gov/browse/DPC-1168)

**Future Work**

N/A

**Checklist**

- [x] All tests are passing via `make ci-app` (app change) and `make ci-web` (website change)
- [N/A] Swagger documentation has been updated
- [N/A] FHIR documentation has been updated
- [N/A] Any required dpc-ops changes have a PR submitted and mentioned in this ticket
- [N/A] Any manual migration steps are documented, scripts written (where applicable), and tested
- [N/A] Before merging, any required dpc-ops changes have been approved and merged into master of the dpc-ops repo

### CODE REVIEW

File: dpc-web/app/views/internal/organizations/show.html.erb [hunk 1]
```diff
@@ -233,6 +233,6 @@
 
 <section class="ds-u-margin--2">
   <div class="ds-l-row ds-u-justify-content--center">
-    <%= link_to "Delete organization", internal_organization_url(@organization), method: :delete, class: "ds-u-color--error ds-u-text-align--center" %>
+    <%= link_to "Delete organization", internal_organization_url(@organization), data: { confirm: 'If it is enabled in an API all data will be permanently deleted. Are you sure you want to delete this organization?' }, method: :delete, class: "ds-u-color--error ds-u-text-align--center" %>
```
Comment (dhgreene) [inline dpc-web/app/views/internal/organizations/show.html.erb:236]: This message confuses me.  Is this not a prompt to delete, not enable?

Comment (Sun-Mountain) [reply dpc-web/app/views/internal/organizations/show.html.erb:236]: It is, but you can still delete an enabled organization, which delete everything in the API. I can run this language by the HC team if you would like.

Comment (dhgreene) [reply dpc-web/app/views/internal/organizations/show.html.erb:236]: I get it now.  @megmcf is this message good for you?

Comment (megmcf) [reply dpc-web/app/views/internal/organizations/show.html.erb:236]: Looks good to me

**Pull request merged**